### PR TITLE
Added isLoggedIn to TPresenter

### PR DESCRIPTION
**Pull request by radekhubner**
Sometimes I need check if user is (not) loggedIn. When I don't call `$this->logIn()` I haven't identity of user.

### CODE REVIEW

File: src/Traits/TPresenter.php [hunk 1]
```diff
@@ -372,6 +372,16 @@ protected function logOut()
 		return $user;
 	}
 
+    /**
+     * @return bool
+     */
+    protected function isLoggedIn()
```
Comment (mrtnzlml) [inline src/Traits/TPresenter.php]: Could you please rename it to `isUserLoggedIn`?

File: tests/Traits/TPresenterTest.phpt [hunk 2]
```diff
@@ -259,6 +259,13 @@ class TPresenterTest extends \Testbench\CustomPresenterTestCase
 		}, 'Tester\AssertException', "field 'test' should be defined as required, but it's not");
 	}
 
+    public function testUserLoggedIn()
+    {
```
Comment (mrtnzlml) [inline tests/Traits/TPresenterTest.phpt]: Please check that user is not logged in at the beginning of the test. Test scenario should be:
- not logged in (test)
- login
- logged in (test)
- logout
- not logged in (test)

**Review by mrtnzlml — CHANGES_REQUESTED** [review]
Also please keep coding standard and use tabs instead of spaces. Thanks...

File: tests/Traits/TPresenterTest.phpt [hunk 3]
```diff
@@ -259,6 +259,15 @@ class TPresenterTest extends \Testbench\CustomPresenterTestCase
 		}, 'Tester\AssertException', "field 'test' should be defined as required, but it's not");
 	}
 
+	public function testUserLoggedIn()
+	{
+		$this->logOut();
```
Comment (mrtnzlml) [inline tests/Traits/TPresenterTest.phpt]: User should be logged out by default (remove this line). Look at the scenario I wrote in previous comment:
- test
- login
- test
- logout
- test

**Comment by radekhubner** [comment]
@mrtnzlml I've edited it.

**Review by mrtnzlml — APPROVED** [review]

**Comment by mrtnzlml** [comment]
Thank you very much @radekhubner.

**Pull request merged**